components/13-ironic/README.md

```diff
@@ -0,0 +1,83 @@
+# OpenStack Ironic
+
+So unfortunately OpenStack Helm doesn't publish helm charts that can be consumed like
+regular helm charts. You must instead clone two of their git repos side by side and
+build the dependencies manually. They additionally don't split out secrets but instead
+template them into giant config files or even executable scripts that then get stored
+as secrets, a clear violation of <https://12factor.net>. As a result we cannot store
+a declarative config of Keystone and allow users to supply their own secrets.
```

Comment:
Is it just because they don't publish them in a repo? If so, could we just host it ourselves internally?

Reply:
That's certainly an option. I'm happy to get some ideas around it.